.gitignore

```diff
@@ -1,5 +1,5 @@
 target/
 bin/
+src-gen/
 .DS_Store
-/org.eclipse.m2e.binaryproject/
```

Comment:
I think this is critical since now changes in project `org.eclipse.m2e.binaryproject` are considered again.
I wonder if the exclusion was intentional?